### PR TITLE
fixes some mem leaks related to font crap I'm working on

### PR DESCRIPTION
**Pull request by RemoveRusky**
rect_packer is still duplicated. I'm not sure who should own it. Perhaps we need a a shared folder. Eitherway, atleast one makefile will need modification to remove one of the copies

### CODE REVIEW

File: CompilerSource/compiler/reshandlers/rectpack.cpp [hunk 1]
```diff
@@ -1,51 +1,55 @@
...
-**  with this code. If not, see <http://www.gnu.org/licenses/>                  **
-**                                                                              **
-**  ENIGMA is an environment designed to create games and other programs with a **
-**  high-level, fully compilable language. Developers of ENIGMA or anything     **
-**  associated with ENIGMA are in no way responsible for its users or           **
-**  applications created by its users, or damages caused by the environment     **
-**  or programs made in the environment.                                        **
-**                                                                              **
-\********************************************************************************/
+/** Copyright (C) 2010-2011 Josh Ventura
```
Comment (JoshDreamland) [inline CompilerSource/compiler/reshandlers/rectpack.cpp]: This file is now identical to the one in the engine, right?

File: ENIGMAsystem/SHELL/Universal_System/fontstruct.cpp [hunk 2]
```diff
@@ -296,42 +290,42 @@ int font_add_sprite(int spr, uint32_t first, bool prop, int sep)
 }
 
 float font_get_glyph_texture_left(int fnt, uint32_t character) {
-  enigma::fontglyph* glyph = findGlyph(enigma::fontstructarray[fnt], character);
+  enigma::fontglyph* glyph = enigma::findGlyph(enigma::fontstructarray[fnt], character);
```
Comment (JoshDreamland) [inline ENIGMAsystem/SHELL/Universal_System/fontstruct.cpp]: This is a source file; you can just say `using enigma::findGlyph;`

File: CompilerSource/compiler/reshandlers/rectpack.cpp (hunk 1 above)
Comment (RemoveRusky) [reply CompilerSource/compiler/reshandlers/rectpack.cpp]: yes

Comment (JoshDreamland) [reply CompilerSource/compiler/reshandlers/rectpack.cpp]: Excellent. Thanks!

**Pull request merged**